amplitude/.gitignore

```diff
@@ -0,0 +1,154 @@
+# Created by http://www.gitignore.io
```

Comment:
Do we need to have a new gitignore? Could we move it to the root folder?

Reply:
Oh, that's okay. But out of curiosity, I've seen some cases where multiple .gitignore's are placed within sub-folders so as to ignore files there...
I understand why it would be best to have a single .gitignore in the root dir